src/client.rs

```diff
@@ -434,7 +438,7 @@ impl ClientState {
                     }
                 } else { debug!("Command failed: Expected room name only (no spaces allowed)"); }
             }
-            "leave" => {
+            "part" | "leave" => {
```

Comment:
Why was "part" added here?

Reply:
Because I kept typing `/part` to leave a channel since I'm used to IRC. Want me to remove it? I know it's unrelated to the PR...

Reply:
All good. The slash-commands will only be used for `cli-client` testing unnecessary after integration is complete.